lib/challenge.js

```diff
@@ -49,9 +61,16 @@ var pet_owner;
   // - make noise (each pet makes its own unique noise, as specified by `noise`.
 
 var some_pet;
-
 // Answer Starts Here
-
+some_pet = {
+  name : "sophie",
+  species : "feline",
+  breed : "calico",
+  noise : "meow"
+  makeNoise : function(){
+    console.log(this.noise);
+  }
+}
```

Comment:
This object literal is syntactically incorrect; the tests (run `grunt test`) would have caught this, but not explained why, and `grunt nag` and your linter will show you exactly what's wrong here.

Bottom line, this is one minor fix from being entirely correct. 1